extensions/wikia/PortableInfobox/templates/PortableInfoboxGallery.mustache

```diff
@@ -0,0 +1,20 @@
+{{#hero}}
+  <figure class="pi-item pi-image pi-image-gallery">
```

Comment:
Please consider reusing image template for example as a mustache partial. That way you will avoid code duplication